docs/csharp/language-reference/operators/true-false-operators.md

```diff
@@ -20,7 +20,7 @@ A type with the defined `true` operator can be the type of a result of a control
 
 The following example presents the type that defines both `true` and `false` operators. Moreover, it overloads the logical AND operator `&` in such a way that the operator `&&` also can be evaluated for the operands of that type.
 
-[!code-csharp-interactive[true and false operators example](~/samples/snippets/csharp/keywords/TrueFalseOperatorsExample.cs)]
+[!code-csharp-interactive[true and false operators example](~/samples/csharp/language-reference/operators/TrueFalseOperators.cs)]
```

Comment:
This is unrelated to this PR:

The current limitation where docs.microsoft.com scaffolds all interactive samples means that this one does not work.  Can you change it from `code-csharp-interactive` to `code-csharp`? 

We can turn in back on when the features to support full programs are delivered.

```suggestion
[!code-csharp[true and false operators example](~/samples/csharp/language-reference/operators/TrueFalseOperators.cs)]
```